### PR TITLE
Update to sylius 1.10

### PR DESCRIPTION
**Pull request by pszost**
GitHub Actions: https://github.com/pszost/SyliusQuadPayPlugin/actions/runs/1161902379

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,140 +1,214 @@
...
+
+
+<div class="item" {{ sylius_test_html_attribute('payment-item') }} {{ isQuadPayMethod ? 'quadpay-method' : '' }}>
+    <div class="field">
+        <div class="ui radio checkbox" {{ sylius_test_html_attribute('payment-method-checkbox') }}>
+            {{ form_widget(form, sylius_test_form_attribute('payment-method-select')) }}
+        </div>
+    </div>
+    <div class="content">
+        {% if isQuadPayMethod %}
```
Comment (er1z) [inline README.md]: It'll be nice if root variables follow snake-case convention.

File: src/Resources/views/_widget.html.twig [hunk 2]
```diff
@@ -6,33 +6,9 @@
...
-        </span>
-
-        <span id="qp-descrip-max-amount" data-max-amount="{{ maxAmount }}" style="{{ amount > maxAmount ? '' : 'display: none;' }}">
-            on orders <span class="qp--nowrap">up to {{ maxAmount|sylius_format_money('USD', sylius.localeCode) }}</span>
-        </span>
-
-        <span id="qp-descrip__price" style="{{ amount < maxAmount and amount > minAmount ? '' : 'display: none;' }}">
-            of <span class="qp-descrip__price">{{ (amount / 4)|sylius_format_money('USD', sylius.localeCode) }}</span>
-        </span>
+        We support QuadPay
```
Comment (er1z) [inline src/Resources/views/_widget.html.twig]: Isn't above changed too far? Don't we need all the descriptions above?

File: tests/Application/config/sylius/1.8/bundles.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
```
Comment (er1z) [inline tests/Application/config/sylius/1.8/bundles.php]: Is this still required if minimal Sylius version is 1.9 specified in composer.json?

File: tests/Behat/Page/External/PaymentPage.php [hunk 4]
```diff
@@ -46,18 +38,18 @@ public function capture(): void
...
 
-    private function findToken(string $type = 'capture'): TokenInterface
+    private function findToken(): TokenInterface
     {
         $tokens = [];
 
         /** @var TokenInterface $token */
         foreach ($this->securityTokenRepository->findAll() as $token) {
-            if (strpos($token->getTargetUrl(), $type)) {
+            if (strpos($token->getTargetUrl(), 'capture')) {
```
Comment (er1z) [inline tests/Behat/Page/External/PaymentPage.php]: const

File: tests/Behat/Resources/services/contexts/ui.yml [hunk 5]
```diff
@@ -5,6 +5,7 @@ services:
             - "@bitbag_sylius_quadpay_plugin.page.admin.payment_method.create"
         tags:
             - { name: fob.context_service }
+        public: true
```
Comment (er1z) [inline tests/Behat/Resources/services/contexts/ui.yml]: If you use `_defaults.public`, you'll not need to publish each service in this file.

File: src/Resources/views/_widget.html.twig (hunk 2 above)
Comment (pszost) [reply src/Resources/views/_widget.html.twig]: The installment calculations are wrong and the whole thing displays badly. This requires rewriting the functionality

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -0,0 +1,142 @@
+name: Build
+
+on:
+    push: ~
+    release:
+        types: [created]
+    schedule:
+        -
+            cron: "0 1 * * 6" # Run at 1am every Saturday
```
Comment (leszczuu) [inline .github/workflows/build.yml]: do We want this to run every week?

File: tests/Application/templates/bundles/SyliusShopBundle/Cart/Summary/_totals.html.twig [hunk 7]
```diff
@@ -0,0 +1,65 @@
+{% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
+
+{% set itemsSubtotal = sylius_order_items_subtotal(cart) %}
```
Comment (leszczuu) [inline tests/Application/templates/bundles/SyliusShopBundle/Cart/Summary/_totals.html.twig]: if You are going to change variables in .twig to snake case - do it here as well items_subtotal

**Review by leszczuu — APPROVED** [review]

File: tests/Application/templates/bundles/SyliusShopBundle/Cart/Summary/_totals.html.twig [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+                <td>{{ 'sylius.ui.shipping_estimated_cost'|trans }}:</td>
+                <td class="right aligned">
+                    {% if cart.getAdjustmentsTotal('shipping') > cart.shippingTotal %}
+                        <span class="old-price">{{ money.convertAndFormat(cart.getAdjustmentsTotal('shipping')) }}</span>
+                    {% endif %}
+                    <span id="sylius-cart-shipping-total" {{ sylius_test_html_attribute('cart-shipping-total') }}>{{ money.convertAndFormat(cart.shippingTotal) }}</span>
+                </td>
+            </tr>
+        {% endif %}
+        <tr {% if tax_included and not tax_excluded %}class="tax-disabled"{% endif %}>
```
Comment (er1z) [inline tests/Application/templates/bundles/SyliusShopBundle/Cart/Summary/_totals.html.twig]: Hint: you can break these expressions to multi-line; `{%-` keeps no wrapping whitespace before and `-%}` after. FYI. :)

**Review by er1z — APPROVED** [review]

**Review by er1z — APPROVED** [review]

**Pull request merged**